torchvision/models/segmentation/segmentation.py

```diff
@@ -22,7 +24,13 @@
 }
 
 
-def _segm_model(name, backbone_name, num_classes, aux, pretrained_backbone=True):
+def _segm_model(
+    name: str,
+    backbone_name: str,
+    num_classes: int,
+    aux: Optional[bool] = None,
```

Comment:
Since the PR is about adding types, I propose not to modify the actual arguments of any of the methods including the private ones:
```suggestion
    aux: bool,
```